guildwars2/daily.py

```diff
@@ -81,13 +87,34 @@ class DailyMixin:
         for category in categories:
             if category == "psna" and datetime.datetime.utcnow().hour >= 8:
                 value = "\n".join(dailies["psna_later"])
+            elif category == "fractals":
+                fractals = self.get_fractals(dailies["fractals"])
+                value = "".join(fractals + "\n")
             else:
                 value = "\n".join(dailies[category])
             if category == "psna_later":
                 category = "psna in 8 hours"
             embed.add_field(name=category.upper(), value=value, inline=False)
         return embed
 
+    def get_fractals(self, fractals):
+        daily_recs = []
+        fractal_final = ""
```

Comment:
Output would probably be better off as a list, then "\n".join() it in the main function

Reply:
Made output into a list and "\n" joined them